sci-astronomy/stellarium/stellarium-0.19.0.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop flag-o-matic gnome2-utils virtualx
```

Comment:
EAPI-7 bump looks easy on this one and `gnome2-utils` should be dropped in favor of `xdg-utils` (same line count) or `xdg` (drops need for postinst and postrm, but needs to make sure that `src_prepare` running `cmake-utils_src_prepare` isn't dropped.